src/pages/edge/adobe-journey-optimizer-decisioning/assurance-for-decisioning.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Assurance with decisioning
+description: An introduction to assurance features with Adobe Journey Optimizer Decisioning.
+keywords:
+- Assurance
+- Validate
+- Review & Simulate
+---
+
+# Assurance
```

Comment:
Let's call this `Validation and Preview using AEP Assurance`

Reply:
No "AEP" - AEP goes against brand guidelines. Make that "... using Assurance".